spark/dl/src/main/scala/com/intel/analytics/bigdl/example/udfpredictor/StructuredStreamPredictor.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intel.analytics.bigdl.example.udfpredictor
+
+import com.intel.analytics.bigdl.utils.Engine
+import org.apache.log4j.{Level => Levle4j, Logger => Logger4j}
+import org.apache.spark.sql.functions._
+import org.slf4j.{Logger, LoggerFactory}
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.types.StructType
+
+
+object  StructuredStreamPredictor {
+
+  private val log: Logger = LoggerFactory.getLogger(this.getClass)
+  Logger4j.getLogger("org").setLevel(Levle4j.ERROR)
```

Comment:
Consider use the new logger utils in this:
https://github.com/intel-analytics/BigDL/wiki/Programming-Guide#logging